README.md

```diff
@@ -252,9 +252,15 @@ with Connector() as connector:
             print(row)
 ```
 
-### Specifying Public or Private IP
+### Specifying IP Address Type
+
+The Cloud SQL Python Connector can be used to connect to Cloud SQL instances
+using both public and private IP addresses, as well as
+[Private Service Connect][psc](PSC). To specify which IP address type to connect
```

Comment:
It looks like maybe this isn't linked correctly.

Reply:
@hessjcg how so? Link works for me, I added a space between (PSC) so it looks visually better, otherwise I think the link is working.